File: URI Solutions/1012.cpp
#include <iostream>
#include<iomanip>
#include<string>

using namespace std;

void print(string out,double value)
{
    cout<<out<<fixed<<setprecision(3)<<value<<endl;
}

int main(int argc, char *argv[])
{
    const double pi = 3.14159;
    double a,b,c;
    while(cin>>a>>b>>c)
    {
        print("TRIANGULO: ",0.5*a*c);
        print("CIRCULO: ",pi*c*c);
        print("TRAPEZIO: ",(a+b)/2*c);
        print("QUADRADO: ",b*b);
        print("RETANGULO: ",a*b);
    }
    return 0;
}

File: URI Solutions/1012.cs
using System;

class URI
{
    static void Main(string[] args)
    {
            double pi = 3.14159, a, b, c;
            string input;

            input = Console.ReadLine();
            a = Convert.ToDouble(input.Split(' ')[0]);
            b = Convert.ToDouble(input.Split(' ')[1]);
            c = Convert.ToDouble(input.Split(' ')[2]);

            Console.WriteLine("TRIANGULO: " + (0.5 * a * c).ToString("0.000"));
            Console.WriteLine("CIRCULO: " + (pi * c * c).ToString("0.000"));
            Console.WriteLine("TRAPEZIO: " + ((a + b) / 2 * c).ToString("0.000"));
            Console.WriteLine("QUADRADO: " + (b * b).ToString("0.000"));
            Console.WriteLine("RETANGULO: " + (a * b).ToString("0.000"));
    }


}
